Dockerfile.rhel

```diff
@@ -0,0 +1,5 @@
+FROM quay.io/operator-framework/ansible-operator:master
```

Comment:
Can we make this the downstream image? it should be available now, and is this the correct place?